pages/faucet/solana.tsx

```diff
@@ -30,9 +30,9 @@ const SolanaFaucet: ThirdwebNextPage = () => {
         mx="auto"
         px={{ base: 0, md: 4 }}
       >
-        <Heading>Solana faucet</Heading>
-        <Heading fontSize="20px" my="4">
-          Get Solana devnet tokens for free
+        <Heading mr={3} as="h1">Solana Faucet</Heading>
```

Comment:
for the extra spacing here `mr={3}` let's instead add `gap={3}` on the `<Flex />` component above (prefer gap spacing over margin spacing)

Reply:
the mr was a copy/paste mistake as Juan told me, he'll fix it